site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -35,25 +38,25 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 ==== Request Body
 
 [.api]
-[field]#displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
-A human readable name for this link. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
+[field]#identityProviderLink.displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
+A human-readable name for this link when initially created. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
```

Comment:
```suggestion
A human-readable value identifying the user in the remote identity provider.
```

Reply:
@mooreds 👍 
"The most valuable of all talents is that of never using a bazillion words when one will do" ~ Thomas Moore Jefferson